stable/azure-devops-agent/templates/service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
does this need a service?

Reply:
Service not needed.  Agent communication is outbound poll only to Azure pipelines.  Will remove.